### PR TITLE
Fix/experiment createria relation

### PR DESCRIPTION
**Pull request by maartenuni**
It was a bit more work that I anticipated, however, this PR will put the one to one relation between Experiment and DefaultCriteria on the Experiment model.
Django didn't like that the Meta class of DefaultCriteriaForm had a field "experiment",as this wasn't displayed anyway, I've removed it for now. Apparently, you may not use back references to Experiment.

### CODE REVIEW

**Comment by bbonf** [comment]
Maybe there's a way to set things up so that `Experiment.objects.create()` doesn't require explicitly creating an empty `DefaultCriteria` object?

File: lab/experiments/migrations/0036_alter_experiment_defaultcriteria.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('experiments', '0035_remove_defaultcriteria_experiment_and_more'),
+    ]
+
+    operations = [
+        migrations.RunPython(select_default_criterion, None),
```
Comment (bbonf) [inline lab/experiments/migrations/0036_alter_experiment_defaultcriteria.py:30]: You've made the effort of making 0035 reversible, but then 0036 isn't. Not too important as the db can be recreated from scratch for now

Comment (bbonf) [reply lab/experiments/migrations/0036_alter_experiment_defaultcriteria.py:30]: Hmm apparently 0035 is also not really reversible as it fails on DefaultCriteria.experiment requiring a non-null value (when reversing RemoveField).
To get it to run you need to go to 0006 and add `null=True` to the experiment field. Thanks Django.

**Comment by maartenuni** [comment]
> Maybe there's a way to set things up so that `Experiment.objects.create()` doesn't require explicitly creating an empty `DefaultCriteria` object?

I think i've addressed this issue in my latest commit

**Review by bbonf — APPROVED** [review]

**Pull request merged**